src/Coordination/KeeperDispatcher.h

```diff
@@ -88,6 +113,10 @@ class KeeperDispatcher
     /// Thread apply or wait configuration changes from leader
     void updateConfigurationThread();
 
+    void readRequestThread();
+
+    void finalizeRequestsThread();
```

Comment:
What does it mean finalize? AFAICS We just move requests from `unprocessed_requests_for_session` to `requests_queue`?